OgreMain/src/OgreNumerics.cpp

```diff
@@ -26,57 +26,59 @@ THE SOFTWARE.
 -----------------------------------------------------------------------------
 */
 
-#include "OgreStableHeaders.h"
```

Comment:
please revert changes to this file. This does not improve code quality, but merely adds noise to the commit

Reply:
did